src/data/plantBotanicNames.ts

```diff
@@ -0,0 +1,32 @@
+export const plantBotanicNames: string[] = [
```

Comment:
why do we have to describe all existing plant names, but don`t do this property as smthing like 
`plantName: string  `?